### PR TITLE
Support up to rails 7 and ruby 3.2

### PR DESCRIPTION
**Pull request by cjcolvar**
Also drop testing on ruby 2.6.

### CODE REVIEW

File: lib/samvera-persona.rb [hunk 1]
```diff
@@ -1,20 +1,17 @@
...
  
- require "samvera-persona/engine"
+require "samvera-persona/engine"
 
 module Samvera
   module Persona
-    class << self
-      mattr_accessor :soft_delete
-      self.soft_delete = true
+    mattr_accessor :soft_delete
```
Comment (orangewolf) [inline lib/samvera-persona.rb:10]: doesn't this change the scope? can you still call Samvera::Persona.soft_delete after this change or do you need to instantiate first?

Comment (cjcolvar) [reply lib/samvera-persona.rb:10]: It worked for me in the test app and in Avalon.  I also added some tests for it: spec/lib/samvera-persona_spec.rb

**Review by orangewolf — CHANGES_REQUESTED** [review]
looks like `spec/lib/.samvera-persona_spec.rb.swp` got added on accident. the rest of this looks great.

**Comment by cjcolvar** [comment]
Gah, sorry about that.  I removed it.

**Review by orangewolf — APPROVED** [review]

**Pull request merged**